infrastructure/resource_group_level/main.bicep

```diff
@@ -55,6 +55,7 @@ param storageNetworkAccessDefaultAction string = 'Allow'
 @description('Enable 7-day soft deletes on file shares')
 param shareDeleteRetentionEnabled bool = false
 
+param cosmosDbDatabaseName string = 'dtfs-submissions'
```

Comment:
The webapp uses cosmosdb and I noticed the database name is configurable, even though all environments use the same name. I've updated the bicep file.